include/dbstructure.php

```diff
@@ -1092,39 +1092,39 @@ function db_definition($charset) {
 			);
 	$database["oembed"] = array(
 			"fields" => array(
-					"url" => array("type" => "varchar(255)", "not null" => "1", "primary" => "1"),
+					"url" => array("type" => "varbinary(255)", "not null" => "1", "primary" => "1"),
```

Comment:
Fair enough, what was your notice in the original PR comment "We should only use "varbinary" at places where we are sure that only 1 byte characters are used." about, then?